### ISSUE TITLE
prepare installation instructions

### ISSUE THREAD

**Issue by HuangJunye** [HuangJunye/GrapheneLab-Measurement-Code#47]

**Comment by HuangJunye** [comment]
add path to source code to jupyter notebook first cell.
```
import sys
sys.path.append(r"C:\Users\Username\Documents\GrapheneLab-Measurement-Code")
```
In this example, the source directory is "C:\Users\Username\Documents\GrapheneLab-Measurement-Code"

**Comment by HuangJunye** [comment]
Implemented by #49
